### ISSUE TITLE
Awareness of Units Summit Summer 2022

### ISSUE THREAD

**Issue by stuchalk** [uncefact/spec-jsonld#89]
Sorry this is not an issue but a message just trying to get in touch with the developers of this project. There will be three coordinated session at ScIDataCon 2022. See this page for more info. https://codata.org/initiatives/task-groups/drum/units-summit-recap-meeting/

Stuart Chalk
DRUM Secretary
https://codata.org/initiatives/task-groups/drum/

**Comment by stuchalk** [comment]
It' both an invite to participate in the summit and tell us what is need to support digital units, and an awareness issue.  There is a lot happening (or about to happen) in this space and the DRUM group is trying hard to make sure the community is aware and can tell us what is needed for digital units…

Stuart

On May 26, 2022, at 4:22 PM, David Roff ***@***.******@***.***>> wrote:



Hi @stuchalk<https://nam10.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fstuchalk&data=05%7C01%7Cschalk%40unf.edu%7C0f46c8011e544f0398e408da3f557178%7Cdf29b2fa8929482f9dbb60ff4df224c4%7C1%7C0%7C637891933459183355%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Yt8NozYhg18ssPn7GVP6vFOxD152VU7890cbrLMHL3I%3D&reserved=0>

thanks for dropping in an issue, I’ve looked at the links and whilst it looks interesting I’m just wondering what the ask is of our vocab team?

Units of measure are one of the UN recommendations and a code list, some work including work from members of this team within the UNECE will make these code lists more available and machine readable. So we will naturally be using the UN recommendations.

Is there something specific or is it to raise awareness of what you are doing also?

Welcome any collaboration from your groups we have call every other week next one tomorrow 10 CET

**Comment by nissimsan** [comment]
Note we're already aligning to https://www.qudt.org/

**Comment by stuchalk** [comment]
Yes, I have seen some of what's going on and that is good news :)

**Comment by stuchalk** [comment]
So, I am working on building out the units of measurement interoperability service and I want to include the new UNECE unit representations in the system (along with the old ones).  What is the best way to get this information plus the equivalent QUDT representations?  Either a file, webservice output, or DB dump whichever you recommend...
